process/interceptors/processor/validatorInfoInterceptorProcessor.go

```diff
@@ -60,7 +74,21 @@ func (viip *validatorInfoInterceptorProcessor) Save(data process.InterceptedData
 
 	viip.validatorInfoPool.HasOrAdd(hash, validatorInfo, validatorInfo.Size())
 
-	return nil
+	return viip.updateStorage(hash, validatorInfo, epoch)
```

Comment:
I think we don't need to put them in the storage. This decision will be taken by the proposer/validator if the received stuff will be used/included in one block. Do we use this approach to the other intercepted objects in this package? If for each intercepted object we will access also the disk the performance would decrease very much in this side, I guess. Maybe @iulianpascalau will have an opinion here.

Reply:
No, this approach is not used in the package. So probably this component should not receive the storer anymore.

Reply:
fixed